components/screens/BlogScreen/BlogScreen.js

```diff
@@ -1,12 +1,68 @@
+import { useIsFocused } from '@react-navigation/native';
 import React from 'react';
-import { View, Text } from 'react-native';
+import { useState, useEffect } from 'react';
+import { View, Text, ScrollView, Linking, Button } from 'react-native';
+import { Image } from 'react-native-elements';
+import InAppBrowser from 'react-native-inappbrowser-reborn';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import { Styles } from './BlogStyles';
+
+var blogData = [];
 
 const BlogScreen = () => {
+
+  const [data, setData] = useState([]);
+  const isFocused = useIsFocused();
+
+  const getData = async () => {
+    const resp = await fetch("https://cascade-api.herokuapp.com/blog");
+    const data = await resp.json();
+    setData(data);
+  };
+  useEffect(() => {
+    if (isFocused) {
+      getData();
+
+    }
+
+  }, [isFocused]);
+
   return (
-    <View>
-      <Text>Blog View</Text>
-    </View>
+    <SafeAreaView>
+      <ScrollView>
```

Comment:
If you are keeping ScrollView here, it'd be fantastic to have a toast saying the info is loading. My screen took a bit to upload, and I would have killed the app if I had not seen Caleb's comment about Flatlist earlier.

Reply:
Thank you for the suggestion, Natalia, the loading notification is ready for the next PR!